Initial support for apache 2.4

kepi opened the pull request:
Original apache-top doesn't work on apache 2.4 because of small
diferences in output. There is new table with thread listing on
top of output which cause troubles.

This initial support just ignore this table so everything works
in same way as in 2.2 and older.

kepi committed:
Initial support for apache 2.4

Original apache-top doesn't work on apache 2.4 because of small
diferences in output. There is new table with thread listing on
top of output which cause troubles.

This initial support just ignore this table so everything works
in same way as in 2.2 and older.

fr3nd commented:
Hi and thanks for your pull request. It looks great.

However, when I try to run it and for some reason it fails to reach the server-status page, the script fails with the following error:

```
Traceback (most recent call last):
  File "./apache-top.py", line 437, in <module>
    stdscr.keypad(0)
NameError: name 'stdscr' is not defined
```

It should show an error message instead.

If you fix this annoyance I'll merge it

kepi commented:
Yeah, I'm no python master and not sure about how nested exception works  :)

Fixed version pushed - I moved whole block to detect apache version before curses initialization which is anyway much better.

fr3nd commented:
I'm testing your changes with Apache 2.4.6 and I'm unable to make it work... What version did you test it with?

kepi commented:
Version 2.4.10 and 2.2.21. Can you send me your status page so I can check what differences are?

altmas5 commented:
Hi @kepi, thanks for the support.
I also got an error testing on Apache version 2.4.16:
```
ERROR parsing the data. Please, make sure you are alowed to read the server-status page and you have ExtendedStatus flag activated
```

This is the html code generated on my mod_status page: http://slexy.org/view/s206tCFDVO

kepi commented:
@altmas5 i'm not able to reach slexy.org. Looks like it doesn't work. Can you upload it somewhere else?

altmas5 commented:
Sorry @kepi,

Try with this one: http://pastebin.com/GYBsJ7UM

kepi commented:
@altmas5 please turn ExtendedStatus On in your config. If you open file you provided me with on pastebin, you can see that on last line there is info you don't have it enabled. Apache-top is working only with extended status.

@fr3nd Can you please provide me with raw output from status page so I can fix this?
